docs/try-viam/rover-resources/rover-tutorial.md

```diff
@@ -250,7 +250,7 @@ The following are just a few ideas, but you can expand or modify the rover kit w
 - For GPS navigation, we support NMEA (using serial and I<sup>2</sup>C) and RTK.
   Make and model don't make a difference as long as you use these protocols.
   See [Movement Sensor Component](/components/movement-sensor) for more information.
-- For [LiDAR laser range scanning](/services/slam/run-slam-cartographer), we recommend Velodyne, or RPLIDAR (including A1, which is a sub-$100 LIDAR).
+- For [LiDAR laser range scanning](/services/slam/cartographer), we recommend Velodyne, or RPLIDAR (including A1, which is a sub-$100 LIDAR).
```

Comment:
[q] This link does not work in the preview mode. Is that a problem?

Reply:
It should work in prod, commit hash doesn't autopopulate in staging with absolute links which breaks it